src/Core/hco_interp_mod.F90

```diff
@@ -578,46 +530,28 @@ END SUBROUTINE ModelLev_Check
 ! !DESCRIPTION: Subroutine ModelLev\_Interpolate puts 3D data from an
 ! arbitrary number of model levels onto the vertical levels of the simulation
 ! grid. Since the input data is already on model levels, this is only to
-! inflate/collapse fields between native/reduced vertical levels, e.g. from
-! 72 native GEOS-5 levels onto the reduced 47 levels. The vertical
-! interpolation scheme depends on compiler switches. If none of the compiler
-! switches listed below is used, no vertical interpolation is performed,
-! e.g. the vertical levels of the input grid are retained.
-!\\
-!\\
+! collapse fields between native/reduced vertical levels, e.g. from
+! 72 native GEOS-5 levels onto the reduced 47 levels.
+!
+!
 ! The input data (REGR\_4D) is expected to be already regridded horizontally.
 ! The 4th dimension of REGR\_4D denotes time.
-!\\
-!\\
+!
+!
 ! The 3rd dimension of REGR\_3D holds the vertical levels. It is assumed that
 ! these are model levels, starting at the surface (level 1). If the input
-! data holds 72 input levels, this is interpreted as native data and will
-! be collapsed onto the reduced grid. If the input data holds X <=47 levels,
-! these levels are interpreted as levels 1-X of the reduced grid. In other
-! words, input data with 33 levels will be interpreted as 33 levels on the
-! reduced grid, and the data is accordingly mapped onto the simulation grid.
-! If data becomes inflated or collapsed, the output data will always extent
-! over all vertical levels of the simulation grid. If necessary, the unused
-! upper levels will be filled with zeros. If no data interpolation is needed,
-! the vertical extent of the output data is limited to the number of used
-! levels. For instance, if the input data has 5 vertical levels, the output
-! array will only extent over those 5 (bottom) levels.
-!\\
-!\\
+! data holds 72/73 input levels, this is interpreted as native data and will
+! be collapsed onto the reduced GEOS-5 grid. If the input holds 102/103 input
+! levels, this is interpreted as native data and will be collapsed onto the 
+! reduced GISS grid (nbalasus, 8/10/2023).
+!
+!
 ! Currently, this routine can remap the following combinations:
-!\begin{itemize}
-! \item Native  GEOS-5 onto reduced GEOS-5 (72 --> 47 levels)
-! \item Reduced GEOS-5 onto native  GEOS-5 (47 --> 72 levels)
-! \item Native  GEOS-4 onto reduced GEOS-4 (55 --> 30 levels)
-! \item Reduced GEOS-4 onto native  GEOS-4 (30 --> 55 levels)
-! \item Native  GEOS-5 onto native  GEOS-4 (72 --> 55 levels)
-! \item Reduced GEOS-5 onto native  GEOS-4 (47 --> 55 levels)
-! \item Native  GEOS-5 onto reduced GEOS-4 (72 --> 30 levels)
-! \item Reduced GEOS-5 onto reduced GEOS-4 (47 --> 30 levels)
-!\end{itemize}
-! Interpolation from GEOS-5 onto GEOS-4 levels is currently not supported.
-!\\
-!\\
+!
+! Native GEOS-5 onto reduced GEOS-5 (72 --> 47 levels)
+! Native GISS onto reduced GISS (102 -> 74 levels) 
```

Comment:
Maybe could note explicitly here that both data on grid centers (third dimension is sized 72) & grid edges (third dimension is sized 73) are now supported.

Reply:
Addressed.